sql/sql_connect.cc

```diff
@@ -687,6 +687,40 @@ static bool login_connection(THD *thd) {
   DBUG_RETURN(0);
 }
 
+static void thd_update_net_stats(bool need_lock, NET *net) noexcept {
```

Comment:
Remove bool need_lock if it's only a single call site
(and replace my assert_mysql_mutex_own if makes sense)